drivers/usb/device/Kconfig

```diff
@@ -95,4 +95,11 @@ config USB_DC_STM32_DISCONN_ENABLE
 	  Say Y if your board uses USB DISCONNECT pin to enable the
 	  pull-up resistor on USB DP.
 
+config USB_DC_STM32_PA11_PA12_REMAP
```

Comment:
I was thinking if this should be done in DT. We could add an optional property without value in the st,stm32-usb.yaml and add it to STM32F0 SoCs on QFN28 and TSSOP20 packages with USB controller. What do you think?